frontend/src/scenes/startups/startupProgramLogic.ts

```diff
@@ -130,6 +130,24 @@ export const startupProgramLogic = kea<startupProgramLogicType>([
         ],
     }),
     selectors({
+        isYC: [() => [(_, props) => props.referrer], (referrer: string | undefined) => referrer === 'yc'],
+        isReferralProgram: [
+            () => [(_, props) => props.referrer],
+            (referrer: string | undefined) => !!referrer && referrer !== 'yc',
+        ],
+        referrer: [() => [(_, props) => props.referrer], (referrer: string | undefined) => referrer],
+        referrerDisplayName: [
```

Comment:
Is there any risk to this just being wrong / weirdly formatted if the url was shared incorrectly?

Reply:
no strong feelings tbh

Reply:
ok going with the above suggestion